src/Button.js

```diff
@@ -13,12 +13,26 @@ class Button extends Component {
   }
 
   componentDidMount() {
-    const { tooltip, tooltipOptions } = this.props;
-    if (
-      typeof $ !== 'undefined' &&
-      (typeof tooltip !== 'undefined' || typeof tooltipOptions !== 'undefined')
-    ) {
-      $(this._btnEl).tooltip(tooltipOptions);
+    if (!M) return;
+
+    const { tooltip, tooltipOptions, fab } = this.props;
```

Comment:
Here if `tooltipOptions` is undefined you will get an error, why not assign a default value:
`const { tooltip, tooltipOptions = {}, fab } = this.props;`